account-abstraction/alchemy_requestPaymasterAndData.yaml

```diff
@@ -53,5 +53,5 @@ components:
             content:
               application/json:
                 schema:
-                  $ref: ../evm_responses.yaml#/alchemy_requestUserOperationSponsorship
-        operationId: alchemy-requestUserOperationSponsorship
+                  $ref: ../evm_responses.yaml#/alchemy_requestPaymasterAndData
+        operationId: alchemy-alchemy_requestPaymasterAndData
```

Comment:
@SahilAujla the operationId should be alchemy-requestPaymasterAndData

Reply:
Oh, thanks for catching this!

Fixed ✅